apps/mobile/src/features/send/send-form/loaders/send-form-btc-loader.tsx

```diff
@@ -25,7 +26,19 @@ export function SendFormBtcLoader({ account, children }: SendFormBtcLoaderProps)
   const { availableBalance, fiatBalance } = useBitcoinAccountTotalBitcoinBalance(accountId);
 
   // Handle loading and error states
-  if (!utxos.length || !feeRates) return null;
+  // if (!utxos.length || !feeRates) return null;
```

Comment:
We will need to improve on this but for now we can at least allow the form to appear and mock out 0 fees

Reply:
Prob fine but will prob want to show a loading state or something later? So, fees just aren't loaded yet but eventually do?